### PR TITLE
feat: add better skeletons on the positions list on `/portfolio-next`

### PR DESCRIPTION
**Pull request by MeLlamoPablo**
This PR adds a new skeleton loader in `PositionSkeleton` that matches the `Position` component:

https://user-images.githubusercontent.com/11708035/217280710-d59ae691-d558-486d-99ca-553e4c61cb1a.mp4

Note: the loading time has been greatly exaggerated in this video.

# FE PR Checklist

- [x] Self-reviewed each line of code, preempted questions about logic or decisions
- [x] Noted any intentional deviations from design
- [x] Tried hard to break it and failed, think like a malicious user trying to find edge cases to break the app
- [x] Tested on supported device sizes
    - [x] iPhone 12 Pro
    - [x] iPad Air
    - [x] 1920x1080 (how to add custom size: [https://umaar.com/dev-tips/51-add-new-device/](https://umaar.com/dev-tips/51-add-new-device/))
- [x] Tested all widths between 1920px wide to 390px wide and confirmed no weird breakages at widths in between
- [ ] (if related to balances) Tested with empty wallet, modest wallet, whale wallet
- [x] (if browser specific logic) Tested on Brave, Firefox, Safari
- [x] (if styling) Tested Light/Dark mode

### CODE REVIEW

**Review by chrisling-dev — CHANGES_REQUESTED** [review]
LGTM after rebasing feat/portfolio-next-layout and remove the goToTrade function

File: apps/frontend/core-dapp/src/features/position/Position.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import { Box, Flex, Grid, Icon, spacingIncrement, Subtitle } from 'prepo-ui'
 import { Tooltip } from 'antd'
 import styled from 'styled-components'
+import Skeleton from 'react-loading-skeleton'
```
Comment (chrisling-dev) [inline apps/frontend/core-dapp/src/features/position/Position.tsx:5]: oh wait let's use the Skeleton component we've built that would consider for dark theme, it's under `src/components/Skeleton.tsx`

**Review by chrisling-dev — CHANGES_REQUESTED** [review]

File: apps/frontend/core-dapp/src/features/position/Position.tsx [hunk 2]
```diff
@@ -149,3 +150,23 @@ export const Position: React.FC<PositionProps> = ({
     </Flex>
   </Flex>
 )
+
+export const PositionSkeleton: React.FC = () => (
+  <Flex justifyContent="start" gap={8} width="100%">
+    <Skeleton circle height={46} width={46} />
```
Comment (chrisling-dev) [inline apps/frontend/core-dapp/src/features/position/Position.tsx:156]: To match the actual logo size

```suggestion
    <Skeleton circle height={48} width={48} />
```

**Review by chrisling-dev — CHANGES_REQUESTED** [review]
LGTM after nit

**Review by XavierEkkel — APPROVED** [review]
LGTM from UI perspective

**Review by chrisling-dev — APPROVED** [review]
LGTM!

**Pull request merged**